fleetctl/unload.go

```diff
@@ -42,6 +42,11 @@ func runUnloadUnit(args []string) (exit int) {
 		return 1
 	}
 
+	if len(units) == 0 {
+		stderr("No correct unit is found.")
```

Comment:
Hi, @carmstrong, thanks for your comments. The scenario is no correct units are found. If some are found, some cannot be found, it's another scenario, we can give another patch.